index.md

```diff
@@ -1,17 +1,20 @@
 ---
-layout: default
+layout: page
+title: Flutter
 ---
-Flutter
-=======
-
 Flutter is a new way to build high-performance, cross-platform mobile apps.
-Flutter is optimized for today's, and tomorrow's, mobile devices. We are
-focused on low-latency input and high frame rates on Android and iOS.
+Flutter is optimized for today's, and tomorrow's, mobile devices. We are focused
+on low-latency input and high frame rates on Android and iOS.
 
-Next steps:
+## Next steps
 
+ - [FAQ](faq)
```

Comment:
It looked odd to me that this list was out of sync with the mav menu. I could not find a way to re-order the nav items - I had wanted to move FAQ to near the end of the menu.

I'll push this down the list, or move to the see also section.